test/integration/fixtures/regression/issue-2286.js

```diff
@@ -0,0 +1,5 @@
+describe('suite', function () {
+  beforeEach(function () { this.skip() })
+  after(function () { console.log('after in suite') })
+  it('test', function () {})
+})
```

Comment:
(to the second:) You mean the one that uses nested suites with code like this in the inner one and a test in the `after` of the outer one?